src/cmake/externalpackages.cmake

```diff
@@ -198,25 +202,29 @@ checked_find_package (TBB 2017
                       PREFER_CONFIG)
 
 checked_find_package (DCMTK VERSION_MIN 3.6.1)  # For DICOM images
-checked_find_package (FFmpeg VERSION_MIN 3.0)
+checked_find_package (ffmpeg
```

Comment:
I'm not 100% sure, but I think CMake is case sensitive and I think the diff here is due to conan generating CMake configs with a package name that differs from upstream. So we might have to figure out a way to support both `FFmpeg` and `ffmpeg`...

Reply:
Confirming CMake variable names are case sensitive https://cmake.org/cmake/help/book/mastering-cmake/chapter/Writing%20CMakeLists%20Files.html#variables

Reply:
I think in this case, I think ffmpeg doesn't generate a cmake config, we have our own src/cmake/modules/FindFFmpeg.cmake, which we could rename to Findffmpeg.cmake (and probably change the variable names therein to match) and have this work? But we're in real trouble if the conan versions ever conflict with the naming of the exported cmake configs from any of the dependencies.